deployments/knack-services-corridor-retiming.yaml

```diff
@@ -0,0 +1,61 @@
+###
+### A complete description of a Prefect Deployment for flow 'Knack Services: Data Tracker Corridor Retiming'
+###
+name: 'Knack Services: Corridor Retiming'
+description: Wrapper ETL for the atd-knack-services docker image 
+             with defined commands for the corridor retiming view in Data Tracker app
+version: 8234eb85e30f1ed0cc5bdaf51a0b8bb2
+# The work queue that will handle this deployment's runs
+work_queue_name: default
+work_pool_name: atd-data-03
+tags: []
+parameters: {}
```

Comment:
I think if you leave the `parameters` parameter blank it will not add default parameters to your flow. I have been putting them here manually but not 100% sure you have to.